README.md

```diff
@@ -1 +1,81 @@
-# safe-core-sdk
\ No newline at end of file
+# Safe Core SDK
+
+Software development kit that facilitates the interaction with the [Gnosis Safe contracts](https://github.com/gnosis/safe-contracts).
+
+## Install
+
+Install the package with yarn or npm:
+
+```bash
+yarn install
+npm install
+```
+
+## Build
+
+Build the package with yarn or npm:
+
+```bash
+yarn build
+npm build
+```
+
+## Documentation
+
+### Getting Started
+
+This is how executing a Safe transaction with off-chain signatures looks like.
+
+1. Create a Safe account with two owners and threshold equal two (this configuration is just an example):
+```js
+import { ethers } from 'ethers')
+import EthersSafe, { makeSafeTransaction } from 'safe-core-sdk'
+
+const provider = ethers.getDefaultProvider('homestead')
+const wallet1 = ethers.Wallet.createRandom().connect(provider)
+const wallet2 = ethers.Wallet.createRandom().connect(provider)
+
+const safe = await safeContract.setup(
```

Comment:
Where is the `safeContract` coming from?

Reply:
How would you represent an existing Safe account here? It doesn't make sense to add all the logic to explain where this is coming from.

Reply:
It is also weird that a direct interaction with the contract is needed before using the SDK. I guess at some point the SDK should allow to create Safes, right? Should this be prioritized?

Reply:
maybe just add a comment like
```js
// Existing Safe address (e.g. create from a Safe created via https://app.gnosis-safe.io
const safeAddress = "0x<some_safe_address>"
```